tests/qunit/js/util.js

```diff
@@ -419,6 +420,12 @@ define(['exports', 'jquery', 'underscore', 'oae.core', 'jquery.properties-parser
                 '/shared/oae/css/oae.base.css': null,
                 '/shared/oae/css/oae.components.css': null,
                 '/shared/oae/css/oae.core.css': null,
+                '/shared/oae/css/oae.discussion.css': null,
+                '/shared/oae/css/oae.emailverification.css': null,
+                '/shared/oae/css/oae.folder.css': null,
+                '/shared/oae/css/oae.group.css': null,
```

Comment:
- All of these live under `ui/css` and not `shared/oae/css`
- `oae.content.css` is missing
- `oae.signup.css` is missing